src/python/gem5/components/boards/kernel_disk_workload.py

```diff
@@ -74,6 +75,8 @@ def get_default_kernel_args(self) -> List[str]:
         * This assumes the Linux kernel is used.
     """
 
+    _is_workload_set = False
```

Comment:
I think it would be better to move this to `abstract_board` with functions to get the value and set it like we do with the `_is_fs` function. This will move the logic to one single place instead of having to write the logic in every set function that sets workloads.